src/features/Common/data/thunks.js

```diff
@@ -0,0 +1,30 @@
+import { logError } from '@edx/frontend-platform/logging';
+import { camelCaseObject } from '@edx/frontend-platform';
+import {
```

Comment:
Please add an extra space here to split this imports.

```suggestion

import {
```